overview/xarray-in-45-min.ipynb

```diff
@@ -36,23 +36,13 @@
     "%config InlineBackend.figure_format='retina'"
```

Comment:
<p>"to certain file formats"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>